plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -408,7 +408,8 @@ type GivenCekCosts = (?cekCosts :: CekMachineCosts)
 type GivenCekReqs uni fun ann s = (GivenCekRuntime uni fun ann, GivenCekEmitter uni fun s, GivenCekSpender uni fun s, GivenCekSlippage, GivenCekStepCounter s, GivenCekCosts)
 
 data CekUserError
-    = CekOutOfExError !ExRestrictingBudget -- ^ The final overspent (i.e. negative) budget.
+    = CaseBuiltinError Text -- ^ 'Case' over a value of a built-in type failed.
```

Comment:
I'm not sure if I should `newtype`-wrap this text into a proper error-type. I feel like this is OK?